modules/redhen_connection/src/Entity/ConnectionType.php

```diff
@@ -143,4 +142,21 @@ public function getEndpointBundles($num) {
 
     return $bundles;
   }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getEndpointEntityTypes($entity_type_id, $bundle) {
+    // If the current entity type is in a connection endpoint.
+    if ($this->getEndpointEntityTypeId(1) === $entity_type_id || $this->getEndpointEntityTypeId(2) === $entity_type_id) {
+      // If the current entity bundle is in a connection endpoint bundle.
+      $bundle1 = reset($this->getEndpointBundles(1));
```

Comment:
This throws a warning because `reset()` takes a reference to a variable, which means you aren't supposed to pass a function into it ("Notice: Only variables should be passed by reference..."). Unfortunately, there's not a great way to still fit this on 1 line, but certainly we don't want to trigger warnings!

Reply:
(since this is boring, I submitted a fix)